src/routes/error.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const router = express.Router();
+
+router.get("*", function(req, res) {
```

Comment:
Please, research and add some links to this way of handling errors.
Also, should this error handling be in a router?
And for the `function`, please use arrow functions :)

Reply:
This is also from the node.js course.
in index.js
`app.use((req, res) => {
  res.status(404).render("error");
});`